pkg/api/labels.go

```diff
@@ -31,6 +31,8 @@ const (
 	ServiceLabel = "com.docker.compose.service"
 	// ConfigHashLabel stores configuration hash for a compose service
 	ConfigHashLabel = "com.docker.compose.config-hash"
+	// ConfigHashLabel stores configuration hash for a compose service
+	VolumeConfigHashLabel = "com.docker.compose.volume-config-hash"
```

Comment:
you could re-use ConfigHashLabel as this label is used to own the config for resource, regardless resource type

Reply:
I would prefer to create a new label with a new function. It is easier for us ti later find the reference to the volume configuration label later in the code.

Reply:
:-1 we only use this in a few places